docs/index.html

```diff
@@ -228,23 +228,23 @@ <h1 class="visually-hidden">
     DrupalCampNYC 2020
 </h1>
 				<div class="deck--content deck-hero--content">
-					
+
 				</div>
 					</div>
 	</section>
 
 </div>
-            
-        
-        
+
+
+
       <div class="field field--name-field-content field--type-entity-reference-revisions field--label-hidden field__items">
               <div class="field__item">    <section class="deck deck-one-column deck--type--deck-1-column deck--view-mode--default">
                                 <div class="deck--content deck-one-column--content">
                       <div class="paragraph paragraph--type--text paragraph--view-mode--default">
-          
-            <div class="text-formatted field field--name-field-body field--type-text-long field--label-hidden field__item"><div class="button-row-4"><a class="btn btn-lg btn-color-1" data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="5ee70d59-c2e1-429e-830e-11db249a4793" href="/program" role="button" title="Program &amp; Schedule">View latest DrupalCampNYC website</a></div>
+
+            <div class="text-formatted field field--name-field-body field--type-text-long field--label-hidden field__item"><div class="button-row-4"><a class="btn btn-lg btn-color-1" data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="5ee70d59-c2e1-429e-830e-11db249a4793" href="https://www.drupalcamp.nyc" role="button" title="Program &amp; Schedule">View latest DrupalCampNYC website</a></div>
```

Comment:
This is the important line with the URL that needs to be patched each time the Tome Static export is run.

The rest of the changes are either:
* the GTM container environment token, or
* cache-busting hash values appended as query strings to CSS and JS asset paths, or
* whitespace fixes (which happened automatically when I edited `index.html` in `vim`).